### DISCUSSION TITLE
file:bg-red-500 is working in localhost ,but not working in production

### DISCUSSION THREAD

**Discussion by nayeem2208** [tailwindlabs/tailwindcss#13761] [Help]
file:bg-red-500 is working in localhost ,but not working in production,
 <input
          type="file"
          className="flex h-9 shadow-lg text-white rounded-lg font-semibold  border border-input bg-red-600 hover:bg-red-800 bg-background px-3 py-1 text-sm  transition-colors file:border-0 file:bg-transparent file:text-foreground file:text-sm file:font-medium placeholder:text-muted-foreground file:shadow-xl file:bg-red-900 hover:file:bg-red-600 file:rounded-lg file:px-4 file:text-white focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 "
          accept=".csv"
          onChange={handleFileChange}
        />

**Comment by serdar-webdev** [comment]
Hi, can you provide the piece of code and tailwind.config.js file?
